annotated_types/__init__.py

```diff
@@ -419,14 +420,18 @@ class DocInfo:  # type: ignore [no-redef]
         documentation: str
         """The documentation string passed to doc()."""
 
-    def doc(
-        documentation: str,
-    ) -> DocInfo:
-        """
-        Add documentation to a type annotation inside of Annotated.
 
-        For example:
+DocInfo = Doc  # backwards compatibility
 
-        >>> def hi(name: Annotated[int, doc("The name of the user")]) -> None: ...
-        """
-        return DocInfo(documentation)
+
+def doc(
+    documentation: str,
+) -> Doc:
+    """
```

Comment:
Instead of defining a whole new function here, let's just alias `doc = Doc` as we did for `DocInfo`.